### PR TITLE
Review the current proposal of fundraising Working Group

### PR DESCRIPTION
**Pull request by sabderemane**
Hi there, according the issue #9, I took the lead of starting a draft proposal on it, happy to have any feedback create discussion in the forum or elsewhere if needed.

Special thanks to @tim-schilling and @StephanieAG for the initial review.

### CODE REVIEW

File: active/fundraising.md
Comment (sabderemane) [inline active/fundraising.md:40]: I'm unsure what is the best way, initially I thought about an email google group but could be something else

File: active/fundraising.md
Comment (sabderemane) [inline active/fundraising.md:44]: I think this decision needs to be made by a group of people, not just me giving a number.

**Comment by priyapahwa** [comment]
Hey @sabderemane,
This looks fantastic. I have a couple of following queries in mind. Please feel free to point me out if I am wrong.
- I believe we should include a standard statement for the value proposition highlighting how WG’s mission resonates with the donors’ values or needs. In fact, keeping the roots intact, we can elaborate on the statement “The web framework for perfectionists with deadlines” itself but organized in a detailed sponsorship deck for easy access.
- Secondly, IMO to achieve milestones, we should gather more ideas on strategies to implement defining some pre-conditions and code of conduct to follow.

File: active/fundraising.md
Comment (thibaudcolas) [inline active/fundraising.md:10]:
```suggestion
- Build and strengthen relationships between Django and funding sources
```

Seems to me like we might as well not just think of donors here?

File: active/fundraising.md
Comment (thibaudcolas) [inline active/fundraising.md:11]: With this one, I’m struggling to see why this is a responsibility of the fundraising group? Raising awareness feels like something some of the other groups could be doing.

File: active/fundraising.md
Comment (thibaudcolas) [inline active/fundraising.md:16]: Rewording so it’s not just about donations (unless you do intend for it to be). I wonder if it’d be worth even being more specific and calling out different types of funding directly.

```suggestion
- Contacting, developing and implementing financial sponsorships with funding sources
- Clarifying and elaborating on the various creative strategies that funding sources of different sizes and types can use to support Django
```

File: active/fundraising.md
Comment (thibaudcolas) [reply active/fundraising.md:40]: I’d expect it’d be more useful to disclose conflicts of interest as part of decision-making on a case by case basis, than upfront?

I’m not sure I see why this needs to be part of the WG charter to start with, it’s important but even for the board I don’t believe we have this so formally defined.

File: active/fundraising.md
Comment (thibaudcolas) [reply active/fundraising.md:44]: I don’t understand why this would be enough of an issue to call out here?

File: active/fundraising.md
Comment (thibaudcolas) [inline active/fundraising.md:60]: I’d expect more regular updates, and most likely a shared dashboard with active fundraising leads or similar CRM-type data. This is quite critical information for the board to be aware of and have a chance to discuss as part of every meeting.

File: active/fundraising.md
Comment (tim-schilling) [reply active/fundraising.md:44]: I think it's fair to be forward thinking about what this may turn into. Some of the larger employers are going to be ideal funding sources. It seems weird to permit the WG the possibility to have a voting bloc for that company.

I think it's wiser to take a more conservative stance from the start. It can be scaled back if it becomes a problem in the future. However, the reverse situation seems messier to untangle.

File: active/fundraising.md
Comment (tim-schilling) [reply active/fundraising.md:11]: I think a natural byproduct of fundraising is raising awareness. It's not the primary goal, but it should be one of them. A sponsor who gives $10,000 without saying anything about Django is not the same as a sponsor who gives $10,000 and tells their communities about the donation and Django. There's a broader picture as to what fundraising could do. Making it clear to the working group that their efforts mean more than a the bottom line could help influence hard decisions.

File: active/fundraising.md
Comment (knyghty) [inline active/fundraising.md:9]: I think this reads fine, but could also be:
```suggestion
- Acquire funding to secure the financial future of Django and support its future growth
```

File: active/fundraising.md
Comment (knyghty) [inline active/fundraising.md:11]: I don't know if "Web Framework" should be capitalised unless we use this phrasing elsewhere.

Comment (knyghty) [reply active/fundraising.md:11]: Visibility also attracts more funding, so I also see value in this.

File: active/fundraising.md
Comment (knyghty) [inline active/fundraising.md:49]: I guess it doesn't take money to make money after all.

File: active/fundraising.md
Comment (knyghty) [inline active/fundraising.md:54]: Not in love with Slack but it is already there. "Slack" should probably be capitalised.

File: active/fundraising.md
Comment (knyghty) [inline active/fundraising.md:55]: I think it's good to set a specific cadence rather than "at least monthly". I think just specifying monthly meetings doesn't exclude ad-hoc meetings from happening when needed or useful.

**Review by knyghty — COMMENTED** [review]
Just some thoughts and nitpicks really, I think it is a good start.

File: active/fundraising.md
Comment (thibaudcolas) [reply active/fundraising.md:11]: Sounds fair!

File: active/fundraising.md
Comment (thibaudcolas) [reply active/fundraising.md:44]: I can see the problem but am not convinced it’s worth having to define this in that clear-cut of a way. Would rather replace this with a line in "future membership" about having individuals from a good diversity of different organizations perhaps. 

The reverse situation – if it comes to that, the board can always decide to shut down a working group (requires a board vote).

**Comment by thibaudcolas** [comment]
@sabderemane the proposal seems to me like it’s in a good state, since you already have lots of members signed up. I see this is currently in draft state technically, is it ready for further review? Or if not is there anything further you’d need to add?

What’s your plan to select the Chair, Co-Chair, Board Liaison? Do you think we can get this fully completed in time for a board vote in the April meeting?

File: active/fundraising.md
Comment (tim-schilling) [reply active/fundraising.md:44]: Could you elaborate on what problems including this would invite?

Comment (thibaudcolas) [reply active/fundraising.md:44]: Well first we have to agree on a X% figure, which to me feels like a distraction compared to other aspects of getting this group up and running. Then I see it being a distraction in the long term too when selecting new members.

For example, right now 50% of the group’s draft membership is associated with Djangonaut Space. If we’re concerned about voting bloc then that’s a problem at least the way that rule is written right now? It seems entirely plausible the group’s membership could have 50% DEFNA members, or Django Girls, or Wagtail; and consider adding one more. All of which wouldn’t be ideal but if it’s the right person at the right time, I’d rather not have to think about a hard and fast "X%".

Comment (tim-schilling) [reply active/fundraising.md:44]: Thank you Thibaud, that helps me understand your perspective. You're right that the way it's written now would be problematic with the proposed members. That needs a change. Though since I'm not a proposed member and one of your concerns is that this is distracting, I'll leave it to the remainder members to voice their opinions.

File: active/fundraising.md
Comment (jacobian) [reply active/fundraising.md:10]: I tend to agree - for example I'd like to think about more broadly going after grants (eg I think we have a good shot at raising money from the [Sovereign Tech Fund](https://www.sovereigntechfund.de/) so thinking more broadly than "donations" seems great to me

File: active/fundraising.md
Comment (jacobian) [reply active/fundraising.md:40]: Yeah, I do think it's fine to leave the specific mechanism of COI disclosure out of the charter -- it can be an operating procedure, not something in the charter. 

That said, I think it might be sufficient to just ask for COI disclosures as part of the application process, in which case you can easily just add that there. Something like this could work:

> When new members self-nominate, they should disclose any potential COIs to the WG. They should also disclose any COIs that might affect a vote before that vote

File: active/fundraising.md
Comment (jacobian) [reply active/fundraising.md:60]: You could copy some language from what I used in #15 -- https://github.com/django/dsf-working-groups/blob/bde7b66cd4204e22a1b7418aa91fb7c5da2c0cb3/active/fellowship.md?plain=1#L60-L68

File: active/fundraising.md
Comment (jacobian) [inline active/fundraising.md:23]:
```suggestion
- Board Liaison: Jacob Kaplan-Moss
```

I think this'll be (subject to board approval)

**Comment by jacobian** [comment]
By and large this is super-fantastic, thanks to everyone for working on this!

I do have several broader comments, none of which necessarily need to be worked into the charter, per se, but are worth writing down and thinking about for a minute.

- First off, cards on the table: broadly speaking, I don't love making volunteers responsible for fundraising. It's just so much work! I'm hella grateful to everyone who wants to help here, but there's super big burnout potential here and I don't love it. So in the long-term, I hope to see the DSF hire a paid fundraiser and/or executive director. If that happens the goals and activities of this group would change substantially, but more importantly this context really colors a lot of my more general comments and goals for this group.

- One major addition I'd like to see added to the charter: I'd like the group to be responsible for documenting its operations and fundraising efforts! Having a written record of "hey we tried this strategy and here's what happened", and "here are all the ideas we've had and haven't tried" would be super useful. In fact, I think developing and maintaining _strategy_ is potentially the way this group could have the most long-term impact. 

- Along those lines, one way this group could operate would be to make recommendations to the board about things to pursue. We're resource constrained too but we _do_ have _some_ firepower! And we can do things like hire consultants/contractors if they'd help raise money. I'm thinking specifically of grant-writing here, but that's probably just one example of ways we could leverage money to make money. I think the board would super-welcome recommendations about which strategies to peruse and how. This may not need to be written explicitly into the charter, but it could.

- Speaking of writing things into the charter: my recommendation to go with a "less is more" approach. Don't worry too much about encoding specific activities etc., those should land in the operating documents I mentioned a couple of bullet points ago. 

I think that's it. Thanks again for all y'all's work, this rocks and I'm very excited.

File: active/fundraising.md
Comment (StephanieAG) [inline active/fundraising.md:44]: A revision of the CIO section based on the above two comment threads, my opinion, and reading the DSF COI policy:
```suggestion
Members must disclose possible conflicts of interest (COI) to the Fundraising Working Group when they first self-nominate as well as if new COIs arise. No more than two members of the Fundraising Working Group may receive compensation from the same organization at any given time. 

In the spirit of aligning with the [Django Software Foundation Conflict of Interest Policy](https://www.djangoproject.com/foundation/conflict-of-interest/), members must disclose possible COIs that might affect a vote before that vote. That member can then choose to either recuse themselves from the vote or disclose all relevant facts regarding the COI. If they disclose all relevant facts, the remaining members will majority vote whether it will constitute a COI without them present. Ties will be brought to the board to decide prior to vote. The resolution of each COI must be recorded in the Fundraising Working Group notes.
```
My main question is: **do members of the Fundraising Working Group legally counted under the definition of an "Interested Person" from the official DSF COI Policy?** If they do, this section should explicitly state that we defer to that policy (and should also probably be simplified to mainly just say something like "Operates under the DSF COI Policy" with a link to that policy page).

File: active/fundraising.md
Comment (StephanieAG) [reply active/fundraising.md:49]: I also think asking the Board for funds as needed is fine for now

File: active/fundraising.md
Comment (StephanieAG) [reply active/fundraising.md:54]: I don't prefer Slack either (my preference would be private Django Forum thread if possible or email). 

I don't want a discussion on the specific communication method to hold up moving the charter forward though — I imagine anything like this can be adjusted pretty easily once we meet and determine what communication methods everyone prefers (aka: I'd use Slack if it makes the most sense for the group).

File: active/fundraising.md
Comment (StephanieAG) [reply active/fundraising.md:55]: I agree

File: active/fundraising.md
Comment (StephanieAG) [inline active/fundraising.md:60]: I like the language Jacob quoted. Took a first stab at customizing it to the FWG below. I kept it high level as I imagine we'd establish a template report outline anyway and update it as needed to include everything the Board needs and wants to know.  

```suggestion
The Fundraising Working Group is required to submit regular reports on fundraising efforts. 
  
Frequency: at least quarterly, with a strong preference for some sort of live "dashboard" (read: Google Sheet) that's updated more regularly for the Board to check any time. 
  
The specifics of what needs to be in the report will change over time and should be an ongoing conversation between the Fundraising Working Group and the Board. It can and should change and evolve over time. It should probably should include things like anticipated next steps on funding opportunities, planned interactions with major donor leads, updates of other activities the group is working on, etc.
 
The Fundraising Working Group is also expected to report any meta-issues or problems to the board. E.g. ongoing strategic questions without answers, problems with DSF infrastructure, emerging issues, and so forth. 
```

PS: I left in the phrase "problems with DSF infrastructure" even though I'm not sure I actually know what kind of thing that is supposed to be describing.

File: active/fundraising.md
Comment (thibaudcolas) [reply active/fundraising.md:44]: Well, first time I hear of this policy! That’s certainly handy to have that all written down 😄 Yes, if this policy is under active use, to me it’s quite clear that a member of a working group is a "member of a committee with governing board delegated powers".

And with that in mind, I’d prefer we defer to that policy without any re-interpretation.

File: active/fundraising.md
Comment (thibaudcolas) [reply active/fundraising.md:60]: Looks good to me, I’d prefer we aim for "at least monthly" as the work of this group is much more time-dependent.

**Comment by sabderemane** [comment]
Hi here, thank you so much to everyone who participate in the review of the PR, it's really helping a lot! I'm really glad some of you have already done a meeting together  💟 (too bad I was travelling)

I'll see to update the PR ASAP.

> the proposal seems to me like it’s in a good state, since you already have lots of members signed up. I see this is currently in draft state technically, is it ready for further review? Or if not is there anything further you’d need to add?

@thibaudcolas I didn't put it ready for review as I considered it's ready to review when it's also ready for board approval, but I can change this if necessary

> What’s your plan to select the Chair, Co-Chair, Board Liaison? Do you think we can get this fully completed in time for a board vote in the April meeting?

@thibaudcolas I didn't planned something yet, I guess I would create a form to allow people to present themselves for the position they want and see to make a vote after that to make sure everyone agree. Board of liaison I think it's mostly a discussion between me, you and @jacobian . _finger crossed for April_ but not fully sure.

**Comment by sabderemane** [comment]
@priyapahwa @thibaudcolas @jacobian @StephanieAG @knyghty for the chair and co-chair https://forms.gle/D7TcgaZpYvbo9TyLA

File: active/fundraising.md
Comment (StephanieAG) [inline active/fundraising.md:42]:
```suggestion
The Fundraising Working Group adheres to the [Django Software Foundation Conflict of Interest Policy](https://www.djangoproject.com/foundation/conflict-of-interest/) and follows all procedures outlined within it, including those related to disclosure, reporting, and resolution of conflicts of interest. 
```

**Review by StephanieAG — COMMENTED** [review]
Looks good. 
Suggested one update to the COI section so it only refers to the full DSF COI policy which is pretty thorough. I don't think the FWG should add/adjust any COI policy since the DSF already has one, and our committee members already count as "interested persons" under it.

This way, there's just a single source of truth for any and all COI processes.

**Comment by StephanieAG** [comment]
> @priyapahwa @thibaudcolas @jacobian @StephanieAG @knyghty for the chair and co-chair https://forms.gle/D7TcgaZpYvbo9TyLA

What are the responsibilities of a "chair" or "co-chair"? 

I'm not clamoring to fill either of these roles, however if others are not interested in filling the roles and the roles need to be filled in order to get this group up and running, I would consider helping just to get us started.

**Comment by thibaudcolas** [comment]
Good question @StephanieAG 😄 For all groups, I’d expect the Chair & Co-Chair to be responsible for:

1. Animating the group’s discussions, particularly in the context of any meetings
2. Managing the logistics of group membership – when someone joins or leaves the group, they’d make sure any access is updated accordingly, handover / onboarding happens, etc
3. Make sure the group’s reporting to the board is working
4. Any other responsibilities defined by the charter (none currently)

None of those tasks have to be done by the Chair / Co-Chair if other group members are keen to help, the Chair/Co-Chair just need to make sure they get done.

File: active/fundraising.md
Comment (cgl) [inline active/fundraising.md:30]: May I be added too please?

**Review by cgl — APPROVED** [review]
It looks good to me 👍🏻

File: active/fundraising.md
Comment (sabderemane) [reply active/fundraising.md]: @thibaudcolas @priyapahwa @jacobian @cgl any opinion?

**Review by thibaudcolas — COMMENTED** [review]
Also looks good to me! With two caveats:

1. I’d like to see a proposed Chair and Co-Chair as we discussed elsewhere
2. There’s no mention of a special way of working with our DSF Treasurer (@jacobian) and Assistant to the Treasurer (Catherine Holmes). That’s fine by me but would appreciate Jacob’s treasurer double checking

**Comment by sabderemane** [comment]
Adding informations from DjangoCon Europe:
- @thibaudcolas proposed to be chair of the working group
- @priyapahwa wanted to be co-chair of the working group
- @mrchrisadams interested to be a member

**Comment by thibaudcolas** [comment]
@sabderemane I’d like to add this to the agenda for tomorrow’s board meeting, so I’ve updated the PR based on your comment and people’s thumbs up. And marked it as ready for review. I hope that’s ok.

**Review by thibaudcolas — APPROVED** [review]
LGTM!

Sully and Mike from Monster Inc give a thumbs up

**Comment by jacobian** [comment]
This looks fantastic to me as well, thanks for all the effort here!

Two tactical notes:

- I've been working on some fundraising work behind the scenes, so once this group is up and running I should sorta debrief on what I've done so far and see what the WG wants to have handed over to the team.
- Catherine (the DSF's board assistant) has often handled most of the communication and tracking of current sponsors, and is a wealth of knowledge. I think it'd be a good idea for the WG to meet with her as well, get some info, and figure out how to leverage her time/skill.

So I think I'd suggest that once this group is fully operational we set up some time for an initial synchronous call to talk through some of this stuff and lay the groundwork for future work.

Super excited about all this!

**Comment by thibaudcolas** [comment]
As of [today’s board meeting](https://www.djangoproject.com/foundation/minutes/2024/jun/13/dsf-board-monthly-meeting/), our working group is **approved**! 😊 Great work @sabderemane @tim-schilling @StephanieAG @priyapahwa @knyghty @cgl @jacobian!

I’ll merge this now, and get back to you in a few days with initial steps to get the group up and running.

**Pull request merged**
